typhos/suite.py

```diff
@@ -462,6 +473,20 @@ def show_subdisplay(self, widget):
                 dock, QtCore.Qt.AlignTop
             )
 
+        self._new_template()
+        if isinstance(widget, TyphosDeviceDisplay):
+            widget.template_changed.connect(self._new_template)
+        return widget
+
+    def _new_template(self, template: Optional[pathlib.Path] = None) -> None:
+        """Hook for when a new template is selected in a sub-display."""
+        if self.parent() is not None:
+            return
```

Comment:
This means that it only takes effect for standalone typhos suites, right? Rather than when it gets embedded in another app like lucid or lightpath? I think this is probably the right call, though I wonder if those apps need to handle their subdisplay widths themselves.

Reply:
Exactly right

I figured that we can assume what the user wants if the suite isn't embedded (`parent = human user`), but should leave it up to the parent widget otherwise.

Reply:
This was my first thought on seeing this PR 👍